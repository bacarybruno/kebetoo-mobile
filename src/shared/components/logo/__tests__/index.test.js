import setupTest from '@app/config/jest-setup'

import Logo from '../index'

const givenLogo = setupTest(Logo)()

it('renders Logo', () => {
  const { wrapper } = givenLogo()
  expect(wrapper.toJSON()).toMatchSnapshot()
})
