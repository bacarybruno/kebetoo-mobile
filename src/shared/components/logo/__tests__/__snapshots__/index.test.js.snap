// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Logo 1`] = `
<Image
  source={
    Object {
      "testUri": "../../../assets/images/logo.png",
    }
  }
  style={
    Array [
      Object {
        "alignSelf": "center",
        "height": 118,
        "resizeMode": "contain",
        "width": "80%",
      },
      undefined,
    ]
  }
/>
`;
